openmetadata-ui/src/main/resources/ui/src/styles/components/react-awesome-query.less

```diff
@@ -248,6 +254,13 @@
   min-width: 180px;
   .ant-select,
   .widget--widget {
-    width: 100% !important;
+    min-width: 100% !important;
+  }
+
+  .ant-select-multiple {
+    .ant-select-selection-item-content {
+      text-overflow: ellipsis;
+      max-width: 200px;
```

Comment:
we can use mixing here to follow DRY approach 
here is the ref. https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedEditor/send-button.less#L16